### ISSUE TITLE
Add rake assets:precompile to documentation

### ISSUE THREAD

**Issue by custombusinesssystems** [excid3/jumpstart#184]
Hey @excid3,

Long time user of Jumpstart, and love it.

Think it would be super helpful to add the rake assets:precompile to the To get started with your new app: part.

**Comment by excid3** [comment]
`assets:precompile` is only used for production. You need to run `bin/dev` for development.

**Comment by custombusinesssystems** [comment]
Appreciate you Chris

interesting, I cant get jumpstart not to error on my local system without it.

**Comment by excid3** [comment]
cssbundling-rails and jsbundling-rails use separate processes to build assets, so you can no longer run just the rails server.

**Comment by custombusinesssystems** [comment]
so when starting a rails server i have forman running, and I have bin/dev what else do i need to run to not have to precompile?

**Comment by excid3** [comment]
You probably need to clear out those precompiled files if you ran that.

**Comment by custombusinesssystems** [comment]
ok, maybe a better question is... when you start a local with jumpstart, whats your process?

**Comment by excid3** [comment]
I just run bin/dev

**Comment by custombusinesssystems** [comment]
Yeah cool, i will have to do some more research, but when you google error starting jumpstart its a thing, and its always the css. https://gorails.com/forum/error-starting-new-jumpstart-app

might be worth a tutorial on how to use bin/dev

**Comment by custombusinesssystems** [comment]
Well if nothing else hopefully this is a great spot for it.... dont start your server with rails s or rails start. simply replace it with bin/dev for applications using cssbundling-rails and jsbundling-rails.

**Comment by excid3** [comment]
That's why I included it in the instructions here:

<img width="513" alt="Screen Shot 2022-04-08 at 7 53 29 PM" src="https://user-images.githubusercontent.com/67093/162550118-7354bf22-42ad-4e56-9d24-66546b3eede7.png">

Just tried it on a brand new app and works fine for me.

**Comment by custombusinesssystems** [comment]
I get it now, and it makes complete sense. 

The issue i had was that when i first ran it it failed. 

I had to watch your https://gorails.com/episodes/esbuild-jsbundling-rails to understand why and add the  "build": "node esbuild.config.js" to the package.json... so by the time I got there... 

sounds pretty dumb but - bin/dev didnt really play into my mind as the way i was starting the server. 

I am not sure how often you get this, but adding "- to start the server", might save you time and a-lot of people a-lot of heart ache.

Regardless, i appreciate your work, and you are a big part of my rails journey.
